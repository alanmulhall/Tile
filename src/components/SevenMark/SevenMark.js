import React from 'react';
import logo from '../../images/logo.png';

const SevenMark = ({ className }) => (
  <img src={logo} className={className} alt="Channel 7" />
);

export default SevenMark;
