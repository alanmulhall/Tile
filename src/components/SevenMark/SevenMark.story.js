import React from 'react';
import { storiesOf } from '@storybook/react';
import centered from '@storybook/addon-centered';
import withTests from '../../utils/withTests';
import SevenMark from './SevenMark';

const SevenMarkStory = () =>
  storiesOf('SevenMark', module)
    .addDecorator(centered)
    .addDecorator(withTests('SevenMark'))
    .add('Default', () => (
      <SevenMark />
    ));

export default SevenMarkStory;
