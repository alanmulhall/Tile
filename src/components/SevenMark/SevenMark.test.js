import React from 'react';
import 'jest-styled-components';
import renderer from '../../utils/rendererThemeProviderWrapper.js';
import SevenMark from './SevenMark';

describe('<SevenMark />', () => {
  it('renders successfullly', () => {
    const tree = renderer(<SevenMark />);
    expect(tree).toMatchSnapshot();
  });
});
