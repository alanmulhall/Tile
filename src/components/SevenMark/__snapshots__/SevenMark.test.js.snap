// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SevenMark /> renders successfullly 1`] = `
<img
  alt="Channel 7"
  src="test-file-stub"
/>
`;
